docs/adr/014-rollover-2.0.md

```diff
@@ -0,0 +1,66 @@
+# ADR 014 - Rollover 2.0
+
+```
+Author: @thomaseizinger
+Date: 20/01/2022
+Status: Active
+```
+
+## Summary
+
+Revise the design of the rollover protocol to:
+
+1. Run at fixed points in time.
+2. Not require approval by either party.
+
+## Context
+
+Currently, rollover is a protocol that is initiated by the taker pretty much on the hour every hour.
+The exact conditions for triggering are that we want the timestamp of the price event that is used for attestation to be 24h in the future.
+Because the oracle attests on the price exactly every hour, we can rollover to a new price event on the hour.
+
+Rollover is coupled to paying fees.
+In particular, the time window until the attestation can be considered as the current funding period.
+For every new funding period, fees are charged depending on the current market.
+In a bull market, shorts pay longs.
```

Comment:
📣 This contradicts this sentence from https://github.com/itchysats/itchysats/discussions/1167:

> The funding rate can be positive or negative. If the funding rate is positive then there is a bullish sentiment in the market. At these times longs pay the shorts.